code/game/atoms.dm

```diff
@@ -57,9 +57,19 @@ var/global/list/ghdel_profiling = list()
 
 	var/arcanetampered = 0 //A looot of things can be
 
+	var/submerging = 0 // if atom is a liquid body for contents inside
```

Comment:
A new variable added to every single atom in the game for just the cryo tubes? There's probably a better way.

Reply:
i can move it up for now

Reply:
submerging is a property of something that "submerges things inside of it", right? IE: The cryo tubes will have this variable, not the objs/mobs inside of it.

Reply:
fair enough, reducing it down to only doing the stuff on this object